dbms/CMakeLists.txt

```diff
@@ -71,7 +71,9 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
             set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-ctad-maybe-unsupported")
         endif ()
     endif ()
-endif ()
+elseif (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
You can also remove `-Wno-shadow-field-in-constructor` for clang.

Reply:
Alexey, can we run the CI before you even waste your time on review? Because if it doesnt pass the CI, no sense to review it